commons/src/test/java/org/eclipse/kapua/commons/configuration/metatype/EmptyTocdTest.java

```diff
@@ -0,0 +1,128 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eurotech and/or its affiliates and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.kapua.commons.configuration.metatype;
+
+import org.eclipse.kapua.qa.markers.junit.JUnitTests;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@Category(JUnitTests.class)
+public class EmptyTocdTest extends Assert {
+
+    @Before
+    public void createInstanceOfClass() {
+
+        assertNotNull(emptyTocd = new EmptyTocd());
+    }
+
+    EmptyTocd emptyTocd;
+
+    //helper list of objects
+    List<Object> objList = Arrays.asList(
+            new EmptyTocd("1", "name"),
+            new EmptyTocd("", ""),
+            new EmptyTocd("2", ""),
+            new EmptyTocd("", "name2"));
+
+    @Test
+    public void emptyTocdTest() {
+        EmptyTocd emptyTocd = new EmptyTocd("49", "name");
+        assertEquals("49", emptyTocd.getId());
```

Comment:
While using `assertEquals`, `assertNotEquals` (and so on) they al support a message linked to the `assert*` as a first argument. It is a good practice to add a message that shows which specific assert has failed.

So in this case:
`assertEquals("tocd.id", "49", emptyTocd.getId());`
would make any failure linked to a meaningful message which can avoid to have to look at the code to check what has failed.